mudlet-mapper.xml

```diff
@@ -7772,6 +7878,82 @@ end</script>
                             <string>mmp logged in</string>
                         </eventHandlerList>
                     </Script>
+                    <Script isActive="yes" isFolder="no">
+                        <name>mmapper_achaea_went_outside_harness</name>
+                        <packageName></packageName>
+                        <script>-- use Stratospheric Harness in Achaea -- PapaGuacamole
+alreadyRun = alreadyRun or false
+
+function mmp.updateIsles()
+-- can be depreciated when crowdmap updated to put these in the right continents by default
+
+	if alreadyRun then return end
+
+	mmp.addcontinent(191,&quot;Eastern_Isles&quot;) -- Clockwork Isle
+	mmp.addcontinent(180,&quot;Eastern_Isles&quot;) -- Colchis, the Island of
+	mmp.addcontinent(181,&quot;Eastern_Isles&quot;) -- Umbrin, the Port of
+	mmp.addcontinent(192,&quot;Eastern_Isles&quot;) -- Zanzibaar
+	mmp.addcontinent(194,&quot;Northern_Isles&quot;) --Ilyrean
+	mmp.addcontinent(206,&quot;Northern_Isles&quot;) --Suliel Island
+	mmp.addcontinent(294,&quot;Northern_Isles&quot;) --Suliel Island (Barricades)
+	mmp.addcontinent(351,&quot;Northern_Isles&quot;) --Karbaz
+	mmp.addcontinent(209,&quot;Western_Isles&quot;) -- Ageiro, the Isle of
+	mmp.addcontinent(162,&quot;Western_Isles&quot;) -- Tapoa Island
+	mmp.addcontinent(207,&quot;Western_Isles&quot;) -- Tuar, the Island of
+
+	alreadyRun = true
+	
+end
+		
+function mmapper_achaea_went_outside_harness()
+
+	local areaid = mmp.getAreaName(mmp.currentroom)
```

Comment:
From the name, `getAreaName` returns a name, not an ID. However, `mmp.oncontinent` needs an ID, so this is not correct. You may want to use `getRoomArea` instead.